lighthouse-cli/performance-experiment/report/injected-script.js

```diff
@@ -30,17 +30,39 @@ window.addEventListener('DOMContentLoaded', _ => {
   rerunButton.style.display = 'inline-block';
 
   rerunButton.addEventListener('click', () => {
-    rerunButton.setAttribute('status', 'running');
-    rerunLighthouse().then(() => {
+    rerunButton.setAttribute('data-status', 'running');
+    rerunLighthouse({blockedUrlPatterns: getUrlPatternsToBlock()}).then(() => {
       location.reload();
+    }).catch(err => {
+      rurunButton.setAttribute('data-status', 'stable');
+      console.log(err);
+    });
+  });
+
+  const blockToggles = document.querySelectorAll('.js-request-blocking__toggle');
+  blockToggles.forEach(toggle => {
+    toggle.addEventListener('click', () => {
+      const requestNode = toggle.parentNode;
+      if (requestNode.getAttribute('data-to-block') === 'true') {
```

Comment:
fixed. Instead of using data URL, changed to `request_block` (for URL to block in the next run) and `request__blocked` (for URL blocked in the previous run). thanks for the suggestion.